Source/WebCore/style/RuleSetBuilder.h

```diff
@@ -39,7 +39,7 @@ class RuleSetBuilder {
 private:
     RuleSetBuilder(const MQ::MediaQueryEvaluator&);
 
-    void addStyleRule(const StyleRuleWithNesting&);
+    void addStyleRule(StyleRuleWithNesting&);
```

Comment:
`addStyleRuleWIthNesting`?

Reply:
In a precedent review, @darinadler commented that we should let the C++ compiler do the dispatching and keep the name clear of the type. I don't have a strong opinion on this (actually I started this patch with `addStyleRuleWithNesting` but then remembered this old comment 😃 )